### ISSUE TITLE
Unit tests fail: android.content.pm.PackageParser$PackageParserException at PackageParser.java:870

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#5228]
```
> Task :app:testBetaDebugUnitTest

fr.free.nrw.commons.review.ReviewControllerTest > testShowNotification FAILED
    java.lang.RuntimeException at ShadowPackageParser.java:69
        Caused by: android.content.pm.PackageParser$PackageParserException at PackageParser.java:870

fr.free.nrw.commons.review.ReviewControllerTest > testGetMedia FAILED
    java.lang.RuntimeException at ShadowPackageParser.java:69
        Caused by: android.content.pm.PackageParser$PackageParserException at PackageParser.java:870

fr.free.nrw.commons.review.ReviewControllerTest > testPublishProgress FAILED
    java.lang.RuntimeException at ShadowPackageParser.java:69
        Caused by: android.content.pm.PackageParser$PackageParserException at PackageParser.java:870

[... same for hundreds of unit tests ...]
```

**Comment by sivaraam** [comment]
This failure is due to bumping the minSdk to 29. It should hopefully go away when we reduce it back to 21 via #5223

**Comment by sivaraam** [comment]
This should be fixed by #5236. Feel free to re-open if you observe this again.
